packages/kotti-ui/source/kotti-field-inline-edit/todo.txt

```diff
@@ -0,0 +1,54 @@
+- Figma desgn: https://www.figma.com/file/qfT3ItoKI6OAXYWF6oOjwR/Inline-Edit
```

Comment:
This file should be removed and any relevant information moved to usage documentation.